srcpkgs/libXft-bgra/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'libXft-bgra'
+pkgname=libXft-bgra
+version=2.3.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="wget pkg-config"
+makedepends="xorgproto libXrender-devel freetype-devel fontconfig-devel"
+short_desc="Library for configuring and customizing font access"
+maintainer="Timmy Keller <tjk@tjkeller.xyz>"
+license="MIT"
+homepage="$XORG_SITE"
+archive="libXft-${version}.tar.bz2"
+distfiles="${XORG_SITE}/lib/${archive}"
+patchfile="https://gitlab.freedesktop.org/xorg/lib/libxft/-/merge_requests/1.patch"
```

Comment:
This is not acceptable for templates. Make it a distfile and `no_extract` it, though that will still break if someone pushes new commits into the PR.